corehq/apps/repeaters/models.py

```diff
@@ -316,8 +310,10 @@ def post_for_record(self, repeat_record):
         payload = self.get_payload_and_handle_exception(repeat_record)
         url = self.get_url(repeat_record)
         try:
-            response = simple_post_with_logged_timeout(self.domain, payload, url, headers=headers,
-                                                       timeout=POST_TIMEOUT, auth=auth)
+            response = simple_post(payload, url, headers=headers, timeout=POST_TIMEOUT, auth=auth)
+        except (Timeout, ConnectionError) as error:
+            log_repeater_timeout_in_datadog(self.domain)
+            raise RequestConnectionError(error)
```

Comment:
I don't think `repeat_record.handle_exception` is called if you move this here. I think it should still get called with this exception.